yt_dlp/extractor/naver.py

```diff
@@ -326,39 +328,41 @@ def _extract_replay(self, show_id, replay_id):
         }, self._extract_video_info(replay_id, vod_info['video_id'], in_key))
 
     def _extract_show_replays(self, show_id):
-        page = 0
+        page_size = 15  # page_size is now a fixed unchangeable value
```

Comment:
```suggestion
        page_size = 15
```
I can see it is being set to a constant...